src/components/rating/Rating.jsx

```diff
@@ -21,25 +25,51 @@ class Rating extends Component {
     if (ratedStarIndex !== rate) {
       onChange(ratedStarIndex);
     }
-  }
+  };
+
+  onMouseEnter = () => {
+    const {active} = this.props;
```

Comment:
If we only access 1 prop here, I'd skip this destructuring here and go for
```js
if (! this.props.active) {
   return;
}
```
it's much more concise and easier to read.